blocks/library/quote/index.js

```diff
@@ -74,12 +74,12 @@ registerBlock( 'core/quote', {
 		);
 	},
 
-	save( attributes ) {
+	save( { attributes } ) {
 		const { value, citation, style = 1 } = attributes;
 
 		return (
 			<blockquote className={ `blocks-quote-style-${ style }` }>
-				{ value && value.map( ( paragraph, i ) => (
+				{ value && wp.element.Children.map( value, ( paragraph, i ) => (
```

Comment:
Maybe, maybe not. Definitely want to avoid a situation where a developer would need to account for varying shapes of the value, if it ever comes to that (`Array.isArray` checks, `React.Children.map` when we know it's just one element, etc).